src/Table/index.jsx

```diff
@@ -156,6 +161,7 @@ Table.propTypes = {
   })).isRequired,
   headingClassName: PropTypes.arrayOf(PropTypes.string),
   tableSortable: PropTypes.bool,
+  tableFixed: PropTypes.bool,
```

Comment:
Should `width` be added to the `shape` prop under `columns`?

Reply:
Also, I understand that `tableFixed` is following the `boolean` `prop` naming convention but `hasFixedColumnWidths` seems like a more accurate description of what effect this `prop` has on the `Table` component.